packages/template-retail-react-app/app/pages/account/wishlist/partials/wishlist-primary-action.jsx

```diff
@@ -55,44 +68,89 @@ const WishlistPrimaryAction = () => {
                 status: 'error'
             })
         }
+
         setIsLoading(false)
     }
 
-    return (
-        <>
-            {isMasterProduct ? (
-                <>
-                    <Button w={'full'} variant={'solid'} onClick={onOpen}>
-                        <FormattedMessage
-                            defaultMessage="Select Options"
-                            id="wishlist_primary_action.button.select_options"
-                        />
-                    </Button>
-                    {isOpen && (
-                        <ProductViewModal
-                            isOpen={isOpen}
-                            onOpen={onOpen}
-                            onClose={onClose}
-                            product={variant}
-                            addToCart={(variant, quantity) => handleAddToCart(variant, quantity)}
-                        />
-                    )}
-                </>
-            ) : (
+    const buttonText = {
+        viewOptions: (
+            <FormattedMessage
+                defaultMessage="View Options"
+                id="wishlist_primary_action.button.view_options"
+            />
+        ),
+        addToCart: (
+            <FormattedMessage
+                defaultMessage="Add to Cart"
+                id="wishlist_primary_action.button.add_to_cart"
+            />
+        ),
+        addAllToCart: (
+            <FormattedMessage
+                defaultMessage="Add All to Cart"
+                id="wishlist_primary_action.button.add_all_to_cart"
+            />
+        )
+    }
+
+    let button
+
+    if (isProductASet) {
+        button = (
+            <Button
+                as={Link}
+                href={`/product/${variant.id}`}
+                w={'full'}
+                variant={'solid'}
+                _hover={{textDecoration: 'none'}}
+            >
+                {buttonText.viewOptions}
+            </Button>
+        )
+        // TODO: create a new set in BM to test this scenario
```

Comment:
It's fine if our main demo instance doesn't have this change yet (we can create a ticket for that) but has this been tested against real data coming back from the API?

Reply:
Yes, it's been tested. I've created the product set in my sandbox and tested it against the API.